packages/graphback-runtime-mongodb/src/MongoDBDataProvider.ts

```diff
@@ -30,11 +33,16 @@ export class MongoDBDataProvider<Type = any, GraphbackContext = any> implements
       delete data[idField.name];
     }
 
+    this.fieldTransforms
```

Comment:
Did we just implemented a generic field transformation engine?

Reply:
It was discussed here: https://github.com/aerogear/graphback/pull/1301#discussion_r430277272

But I have messed up naming the types and hence the result.

Reply:
This is amazing. So technically this can be used for different reasons - for example we can hook field name into it and do CAMEL_CASE transformation per type or each individual field. You have implemented base for https://github.com/aerogear/graphback/issues/630

![image](https://user-images.githubusercontent.com/981838/83152135-5adbb000-a0f5-11ea-9e28-9427536ca896.png)

Reply:
Happy that I helped 😄